doc/shared/sundials/Install.rst

```diff
@@ -2112,3 +2112,68 @@ Table of SUNDIALS libraries and header files
    |                              |              +----------------------------------------------+
    |                              |              | ``kinsol/kinsol_ls.h``                       |
    +------------------------------+--------------+----------------------------------------------+
+
+
+Installing SUNDIALS on HPC Clusters
+-----------------------------------
+
+.. _Installation.HPC:
+
+This section is a guide for installing SUNDIALS on specific HPC clusters.
+In general, the procedure is the same as described previously for Linux machines.
+The main differences are in the modules and environment variables that are specific
+to different HPC clusters. We aim to keep this section as up to date as possible,
+but it may lag the latest software updates to each cluster.
+
+Frontier
+^^^^^^^^
+
+`Frontier <https://www.olcf.ornl.gov/frontier/>`_ is a Exascale supercomputer at the Oak Ridge
+Leadership Computing Facility. If you are new to this system, then we recommend that you review the
+`Frontier user guide <https://docs.olcf.ornl.gov/systems/frontier_user_guide.html>`_.
+
+**A Standard Installation**
+
+Clone SUNDIALS:
+
+```bash
+git clone https://github.com/LLNL/sundials.git && cd sundials
+```
```

Comment:
```suggestion
.. code-block:: bash

   git clone https://github.com/LLNL/sundials.git && cd sundials
```
```

Reply:
I forgot this was not markdown clearly :)